app-emulation/crun/crun-0.10.4.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit eutils python-any-r1
+
+DESCRIPTION="A fast and low-memory footprint OCI Container Runtime fully written in C"
+HOMEPAGE="https://github.com/containers/crun"
+SRC_URI="https://github.com/containers/${PN}/releases/download/${PV}/${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="bpf +caps doc seccomp systemd static-libs"
+
+DEPEND="
+	dev-libs/yajl
+	sys-libs/libseccomp
+	caps? ( sys-libs/libcap )
+	seccomp? ( sys-libs/libseccomp )
+	systemd? ( sys-apps/systemd )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=dev-lang/python-3.5.0
+	doc? ( dev-go/go-md2man )
+"
+
+DOCS=( README.md )
+
+src_configure() {
+	econf \
```

Comment:
This is not wrong and you can leave it if you want, but just letting you know there's a more foolproof and more prettier array form usable.